docs/python_docs/themes/mx-theme/mxtheme/header_top.html

```diff
@@ -19,6 +19,21 @@
         <a class="page-link" href="{{theme_relative_url}}ecosystem">Ecosystem</a>
         <a class="page-link page-current" href="{{theme_relative_url}}api">Docs & Tutorials</a>
         <a class="page-link" href="https://github.com/apache/incubator-mxnet">GitHub</a>
+        <div class="dropdown">
+          <span style="color:#FFFFFF">master</span>
```

Comment:
+1 to Sheng's comment. 
I remember someone recommending "Document Version" like in Apache Spark and other open source project. Does that fit here? @szha @aaronmarkham

Reply:
Thanks for the feedback Sandeep! I'd be happy to change the dropdown to follow this format if that is the general consensus.